cedar-policy/src/api.rs

```diff
@@ -2110,13 +2247,37 @@ pub struct Context(ast::Context);
 
 impl Context {
     /// Create an empty `Context`
+    /// ```
+    /// use cedar_policy::Context;
+    /// let c = Context::empty();
+    /// // let request: Request = Request::new(Some(principal), Some(action), Some(resource), c);
+    /// ```
     pub fn empty() -> Self {
         Self(ast::Context::empty())
     }
 
     /// Create a `Context` from a map of key to "restricted expression",
     /// or a Vec of `(key, restricted expression)` pairs, or any other iterator
     /// of `(key, restricted expression)` pairs.
+    /// ```
+    /// use cedar_policy::{Context,RestrictedExpression};
```

Comment:
Same comment here about spaces in imports; can we run `cargo fmt`?

Reply:
`cargo fmt` did not fix the space. Updated.